### DISCUSSION TITLE
FA Icons not displayed

### DISCUSSION THREAD

**Discussion by Vladislavvk1337** [requarks/wiki#4731] [Error / Bug Report]
**Describe the bug**
Icons are not properly displayed when using Font Awesome 4 or 5 in the Navigation.
only mdi works!

**Reproduce**
1. Go to Admin then Navigation
2. Create a new link or edit and existing link
3. Choose one icon : "fas fa-git", "fas fa-jira", etc.
At this example: "fa fa-rss" , "fas fa-rss", "mdi-rss" 
4. Displayed nothing instead of the icon, although the icon is part of the Free package.

**Screenshots**
![1](https://user-images.githubusercontent.com/75780600/143501587-8ecc3f03-5d76-4747-b6e5-37fb69db1e31.PNG) ![2](https://user-images.githubusercontent.com/75780600/143501589-ba9b98c0-c0fe-4478-9039-9186c4ddd3ac.PNG)

**INFO**
Build in 3 Docker Container
- Image: requarks/wiki:2
- Image: requarks/wiki-update-companion:latest
- Image: postgres:11 

Container works fine

**Specs**
OS: Ubuntu 20.04
Wiki.js:2
Database: postgres:11

**Comment by BestLeaks** [comment]
I wonder if this will ever be fixed?

**Reply by BestLeaks** [reply]
Any information on this?

**Reply by Vladislavvk1337** [reply]
Nope, no Support or solutions

**Comment by mlwilkerson** [comment]
I just tried this on a sandbox instance I recently set up using Wiki.js 2.5.276 and found that both `fas fa-rss`  and `fa fa-rss` work and produce the expected results when configured to use FA 5. And `fa fa-rss` produces the expected result when configured to use FA 4.

My site was built on Digital Ocean using the [pre-built image](https://docs.requarks.io/install/digitalocean) in the marketplace there. That's maybe not identical to your situation @Vladislavvk1337, but I have verified that this Digital Ocean installation uses the standard docker images: `requarks/wiki:2`  and `requarks/wiki-update-companion:latest`. So it seems pretty similar, at least.

I do notice when inspecting the DOM that Wiki.js is loading the Font Awesome assets from CDN urls. So if you were running this locally while not connected to the internet, it definitely wouldn't be able to load those resources. But without looking at the source code, I'm not sure what other scenarios might result in the icons not being visible.

Do you see any failed Network request in the browser's Network console for these assets?
